mrp_bom_current_stock/wizard/bom_route_current_stock.py

```diff
@@ -47,8 +61,9 @@ def _prepare_line(self, bom_line, level, factor):
             'bom_level': level,
             'product_qty': bom_line.product_qty * factor,
             'product_uom_id': bom_line.product_uom.id,
```

Comment:
I didn't review everything yet, but :
`AttributeError: 'mrp.bom.line' object has no attribute 'product_uom'`
I suppose you should use `bom_line.product_id.uom_id` here.